package.json

```diff
@@ -29,6 +29,7 @@
     "@galacticcouncil/bridge": "0.0.7",
     "@galacticcouncil/math-lbp": "^0.1.3",
     "@galacticcouncil/math-liquidity-mining": "^0.1.3",
+    "@galacticcouncil/math-stableswap": "../HydraDX-wasm/packages/stableswap/",
```

Comment:
There is a math-stableswap released pkg. Can you use it instead?

Reply:
Waiting for new version to be released. Latest is `0.1.4` which doesn't include changes I need.